app/src/lumen/res/layout/forget_tab.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please, do not add `RelativeLayout`s

Reply:
@spacifici I copied it from the cliqz variant. Should I change the original layout file as well?

Reply:
I refactored both the layouts to use `LinearLayout` as the root.